index.html

```diff
@@ -206,10 +206,11 @@ <h1>
     <footer class="footer">
       <ul class="social-icons">
         <li>
-          <a rel="noopener noreferrer" href="https://youtube.com/@alyatalla" target="_blank"><i
-              class="fa-brands fa-youtube"></i></a>
+          <a rel="noopener noreferrer" href="https://www.youtube.com/@alyatalla/" target="_blank"><i
+              class="fa-solid fa-envelope"></i></a>
         </li>
         <li>
+
```

Comment:
- [ ] [Optional] You are designing the mobile version so in the mobile version you will find a horizontal black line at button part. So I would suggest please remove it. It is just like logo of wifi on design.
### Your design

![Screenshot 2023-06-06 210553](https://github.com/AlyAtalla/AlyAtalla/assets/81727988/1b322627-3fb3-4569-84d7-827a093fc2c8)